### PR TITLE
Group Add to search Page

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

Fixes PR comments on https://github.com/keybase/client/pull/3167

and adds user search group view after you click on a user.

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits be0febafae51...1a64ba877132 introduced some visual changes:
 * Changed: **Request_Invite-Start** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466722672296-be0febafae51-1a64ba877132/Request_Invite-Start-be0febafae51.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466722672296-be0febafae51-1a64ba877132/Request_Invite-Start-1a64ba877132.png)  
   Request_Invite-Start comparison

File: shared/actions/search.js [hunk 1]
```diff
@@ -110,36 +123,19 @@ export function search (term: string, platform: SearchPlatforms = 'Keybase') : T
       },
     })
 
-    // TODO daemon rpc, for now api hit
```
Comment (chrisnojima) [inline shared/actions/search.js]: we still want this TODO here. we dont' want to hit this raw endpoint

File: shared/search/index.js [hunk 2]
```diff
@@ -40,15 +30,27 @@ class Search extends Component<void, Props, void> {
...
+     onClickResult: user => { dispatch(addUserToGroup(user)) },
+     selectedService: searchPlatform,
+     onSearch: term => { dispatch(search(term, searchPlatform)); dispatch(hideUserGroup()) },
+     onClickService: platform => { searchPlatform !== platform && dispatch(selectPlatform(platform)) },
+     showUserGroup,
+     selectedUsers,
+     onRemoveUserFromGroup: user => { dispatch(removeUserFromGroup(user)) },
+     onClickUserInGroup: user => { dispatch(selectUserForInfo(user)) },
+     onAddAnotherUserToGroup: () => { dispatch(hideUserGroup()) },
+     onOpenPrivateGroupFolder: () => { console.log('TODO open private group') },
```
Comment (chrisnojima) [inline shared/search/index.js]: why not dispatch the action here? we have the open in kbfs action already, and you know what the path is

File: shared/search/render.desktop.js [hunk 3]
```diff
@@ -3,31 +3,55 @@ import React, {Component} from 'react'
...
 
 class Render extends Component<void, Props, void> {
   _renderComingSoon () {
     return <SearchHelp username={this.props.username} />
   }
 
+  _renderInfoPane () {
+    // TODO(mm) maybe shell out to a smart component here?
+    // One that only takes a username and can figure out the rest
+    return <Box style={{flex: 1}} />
```
Comment (chrisnojima) [inline shared/search/render.desktop.js]: yes

File: shared/actions/search.js (hunk 1 above)
Comment (MarcoPolo) [inline shared/actions/search.js]: 👍

File: shared/search/index.js (hunk 2 above)
Comment (MarcoPolo) [inline shared/search/index.js]: Will do in next PR!

File: shared/search/render.desktop.js (hunk 3 above)
Comment (MarcoPolo) [inline shared/search/render.desktop.js]: Cool, already doing that in the next PR for info pane

**Comment by chrisnojima** [comment]
👍

**Comment by chrisnojima** [comment]
already has thumbs

**Pull request merged**